Simplify CSS for displaying answer feedback

alexsanford opened the pull request:
Fixes #1685 

Previously, answer feedback for multiple choice and true/false questions
was being displayed alongside the answer choices. However, that was
causing text to overlap in some cases.

Now, we simply display the answer feedback under the question for all
question types.

~~This is not quite as pretty for the typical case, so I'm not sure if it's the solution we want to run with. But it's a proposal for one fairly simple way to solve the issue.~~ It's prettier now 😄 

@pgk would you mind taking a look at this? It's my first Sensei PR 🎉 so I appreciate your feedback! 😄 

## Questions with long descriptions

### Before

<img width="1818" alt="screen shot on 2017-09-07 at 16 40 18" src="https://user-images.githubusercontent.com/842193/30181906-480293f0-93eb-11e7-9e2c-082bdb13720c.png">

---

### After

<img width="1790" alt="screen shot on 2017-09-08 at 15 35 39" src="https://user-images.githubusercontent.com/842193/30226160-6897ca8e-94ab-11e7-93b9-87fce81ff332.png">

---

## Questions without long descriptions (i.e. the ones that worked fine before)

### Before

<img width="1788" alt="screen shot on 2017-09-07 at 16 41 04" src="https://user-images.githubusercontent.com/842193/30181925-5d87e0c2-93eb-11e7-8c6b-5a33b347885a.png">

---

### After

<img width="1788" alt="screen shot on 2017-09-08 at 15 36 10" src="https://user-images.githubusercontent.com/842193/30226181-77fb550e-94ab-11e7-9c27-4a8b74136384.png">

dbtlr commented:
eh, I don't really like the new style as well, with the feedback above the answers. Shouldn't it remain paired with the information bubble?

alexsanford commented:
@dbtlr The previous style wasn't actually paired with the info bubble (as you can see on the "Before" for questions with long descriptions). It just happens to look like that in my second example 🙂

The old CSS actually places the feedback text vertically halfway between the top and bottom of the whole question block (from the top of the question text, to the bottom of the answers and notes). That's why some cases had overlap and others didn't.

Although I like the idea of displaying it just under the answers, and just above the notes... Maybe that's a better alternative?

donnapep commented:
> Although I like the idea of displaying it just under the answers, and just above the notes... Maybe that's a better alternative?

That might look better. And it would also look better left-aligned with the radio buttons and info box so it's not hanging off by itself in no man's land. 😁

dbtlr commented:
I agree on code, but I also think @donnapep hit the nail on the head for the best "non-design" implementation :)

alexsanford commented:
Thanks for the comments!

I've updated the position of the message, and updated the screenshots in the description.

What do you think, @donnapep ?

donnapep commented:
Yup, looks a lot better! 👍
